src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -28,6 +30,14 @@ public static void printLongestWordTop100() throws IOException {
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
         // TODO 이 부분에 구현한다.
+
+        words.stream()
+                .filter(word -> word.length() > 12)
+                .distinct()
+                .sorted(Comparator.comparing(String::length).reversed())
```

Comment:
```suggestion
                .sorted((word1, word2) -> Integer.compare(word2.length(), word1.length()))
```
이렇게도 표현할 수 있을 것 같아요 😊